server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1703,7 +1703,7 @@ public PoolDTO createPool(@PathParam("owner_key") @Verify(Owner.class) String ow
         pool.setUpstreamPoolId(inputPoolDTO.getUpstreamPoolId());
         pool.setSubscriptionSubKey(inputPoolDTO.getSubscriptionSubKey());
         pool.setSubscriptionId(inputPoolDTO.getSubscriptionId());
-        pool.setAttribute(Pool.Attributes.CUSTOM_POOL, "true");
+        pool.setLocked(false);
```

Comment:
Since the default is false, this will basically no-op in the majority of cases. What we should do is, instead, set the locked flag to true during refresh (which also covers manifest import), as that's where non-custom pools are coming in anyway.

Reply:
Agreed.

Locked pools will come into play during Import manifest and Scheduled pool Job Flow.
Both involve creation of master pool that will be a common point to set the flag.
This will be taken care of in next commit.